internal/nftables/bee-nftables.conf

```diff
@@ -1,4 +1,11 @@
 define listen_ip = "%s"
+define if_wireguard = "wireguard0"
+define connmark_local = 0x9
+define mark_bees = 0x10
```

Comment:
Maybe add a comment that this mark is set by the go code

Reply:
If I understand correctly, this is to signify that a packet has originated on a honeypot. Would `mark_honeypots_outgoing` or something similar maybe better explain what this does?